shared/teams/confirm-modals/confirm-kick-out.tsx

```diff
@@ -84,38 +84,45 @@ const ConfirmKickOut = (props: Props) => {
     }
   }, [waiting, wasWaiting, onCancel])
 
-  const prompt = `Kick ${membersStr(members)} out of ${teamname}?`
+  const prompt = (
+    <Kb.Text center={true} type="Header" style={styles.prompt}>
+      Kick {membersStr(members)} out of {teamname}?
+    </Kb.Text>
+  )
   const header = (
     <Kb.Box style={styles.positionRelative}>
       <Kb.AvatarLine usernames={members} size={64} layout="horizontal" maxShown={5} />
-      <Kb.Icon
-        boxStyle={members.length <= 5 ? styles.iconContainerSingle : styles.iconContainer}
-        type="iconfont-block"
-        style={styles.headerIcon}
-        sizeType="Small"
-      />
+      <Kb.Box2 direction="horizontal" centerChildren={true} style={styles.iconContainer}>
```

Comment:
This is finnicky - did you test with 1,2, and many?

Reply:
Not many enough actually. Fixed.